zuul.d/edpm_multinode.yaml

```diff
@@ -123,49 +123,6 @@
       - ^.*/*.md
       - ^OWNERS
       - ^.github
-
-- job:
-    name: podified-multinode-galera-deployment-crc
-    parent: podified-multinode-edpm-deployment-crc
-    nodeset:
-      nodes:
-        - name: controller
-          label: cloud-centos-9-stream-tripleo-vexxhost
-        - name: crc
-          label: coreos-crc-extracted-xxl
     vars:
-      cifmw_deploy_edpm: false
-      podified_validation: true
-      cifmw_run_tests: false
       make_openstack_deploy_params:
```

Comment:
No one is using the openstack_deploy target, so this parameter (important, parameter, not env variable) is not gonna be used. Anyway, I don't like to put here explicit role variables from install_yamls, as we are making the jobs "aware" of the internal calls to install_yamls that we are using, that can change at any time without notice and should't brake. For this purpose we should use `cifmw_install_yamls_vars`.

Reply:
updated - thanks. I'm lost with all of those parameters :]